COLLECTOR_PARAMS.md

```diff
@@ -5,7 +5,7 @@
 ### `se`
 - contains the b64 url encoded string of the JSON that was sent via `liveConnect.push` function
 ### `duid`
-- contains the LiveConnect managed first party identifier, in the `${apexDomainHash}--${ULID}`, or `${iframeDomainHash}-${websiteUrlDomainHash}--${ULID}` format
```

Comment:
Our of interest: why did the `${iframeDomainHash}-${websiteUrlDomainHash}--${ULID}` part go?

Reply:
Imagine that there's an iframe in two pages:
```
nike.com
 --> googletackingbla.com
     --> our js

adidas.com
 --> googletackingbla.com
     --> our js
```
In the old setup, the cookies are `nike.com--googletackingbla.com--theid1` and `adidas.com--googletackingbla.com--theid2`. The cookies are different for the same user. Though it's possible to have a cross-site cookie via the iframe `googletackingbla.com--theid`, cause the iframe url is always the same. 

This is my understanding.